django_inlinecss/templatetags/inlinecss.py

```diff
@@ -22,12 +22,13 @@ def render(self, context):
             path = expression.resolve(context, True)
             if path is not None:
                 path = smart_unicode(path)
-            if settings.DEBUG:
+            if settings.DEBUG or not getattr(settings, 'INLINECSS_USE_STATIC_FINDER', False):
                 expanded_path = finders.find(path)
+                css_file = open(expanded_path)
             else:
-                expanded_path = staticfiles_storage.path(path)
+                css_file = staticfiles_storage.open(path)
```

Comment:
Why not rely on try/except here, rather than have a setting? Seems like it would be a simpler approach.

Reply:
it is from 5 years ago 😅 , I think we can close it